### ISSUE TITLE
About how to get the weight

### ISSUE THREAD

**Issue by Hyfred** [jantic/DeOldify#56]
Hello, I want to ask how to download colorize34_gen_192.h5 and bwdefade3_gen_160.h5? Where could I find the file"data/imagenet/ILSVRC/Data/CLS-LOC/train". Extremely grateful

**Comment by jantic** [comment]
That specific set of weights for colorizer aren't available, but this older proven set is:  https://www.dropbox.com/s/7r2wu0af6okv280/colorize_gen_192.h5 .  Note the name difference.  The defader weights aren't yet available because I haven't zeroed in on a model I'm happy with yet.

**Comment by Hyfred** [comment]
Thanks for your help！ @jantic ，I am a little confuse about the different of "ColorizeTraning" and "Defadetraning".  I have checked the code. They look the same. Do they have different scenarios?

**Comment by jantic** [comment]
Yeah, they're actually using the same exact model.  The only difference is the augmentation used to alter the input photos for training.  For colorizer you'll see this in the notebook:

x_tfms = [BlackAndWhiteTransform()]

Which changes the input photos from color to black and white (target photos remain the same- color).

And DeFade uses photos as input altered by this augmentation:

x_tfms = [RandomLighting(0.5, 0.5)]

Originally I was just going to have DeFade do color photo defade (that is, have it process after the black and white to color transform is done by the colorizer). But the more I thought about it, I started thinking that it might be more useful to defade black and white photos first before trying to colorize them.  So I introduced this transform which turns both the input and target photos into black and white for training in DeFade:

extra_aug_tfms = [BlackAndWhiteTransform(tfm_y=TfmType.PIXEL)]

Does that help?
